src/images/google_cloud_sdk.rs

```diff
@@ -11,60 +10,65 @@ pub const DATASTORE_PORT: u16 = 8081;
 pub const FIRESTORE_PORT: u16 = 8080;
 pub const PUBSUB_PORT: u16 = 8085;
 
-const DEFAULT_DATASTORE_PROJECT: &str = "test";
-
-#[derive(Debug, Clone, Default)]
+#[derive(Debug, Clone)]
 pub struct CloudSdkArgs {
     pub host: String,
     pub port: u16,
     pub project: String,
-    pub emulator: Emulator,
+    pub emulator: Option<Emulator>,
+}
+
+impl Default for CloudSdkArgs {
+    fn default() -> Self {
+        Self {
+            host: HOST.to_owned(),
+            port: DEFAULT_PORT,
+            project: String::default(),
+            emulator: None,
+        }
+    }
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Copy)]
 pub enum Emulator {
     Bigtable,
     Datastore,
     Firestore,
     PubSub,
-    Help,
-}
-
-impl Default for Emulator {
-    fn default() -> Self {
-        Emulator::Help
-    }
 }
 
 impl IntoIterator for CloudSdkArgs {
     type Item = String;
     type IntoIter = ::std::vec::IntoIter<String>;
 
     fn into_iter(self) -> <Self as IntoIterator>::IntoIter {
-        let mut args: Vec<String> = ["gcloud", "beta", "emulators"]
-            .iter()
-            .map(|&s| s.to_owned())
-            .collect();
-        args.push(
-            match self.emulator {
-                Emulator::Bigtable => "bigtable",
-                Emulator::Datastore => "datastore",
-                Emulator::Firestore => "firestore",
-                Emulator::PubSub => "pubsub",
-                Emulator::Help => "--help",
-            }
-            .to_owned(),
-        );
-        args.push("start".to_owned());
-        if self.emulator != Emulator::Help {
-            if self.emulator == Emulator::Datastore {
+        if let Some(emulator) = self.emulator {
+            let mut args: Vec<String> = ["gcloud", "beta", "emulators"]
+                .iter()
+                .map(|&s| s.to_owned())
+                .collect();
+            args.push(
+                match emulator {
+                    Emulator::Bigtable => "bigtable",
+                    Emulator::Datastore => "datastore",
+                    Emulator::Firestore => "firestore",
+                    Emulator::PubSub => "pubsub",
+                }
+                .to_owned(),
+            );
+
+            args.push("start".to_owned());
```

Comment:
This could just be a `vec!` no?

```suggestion
			let mut args = vec![
				"gcloud".to_owned(),
				"beta".to_owned(),
				"emulators".to_owned(),
				match emulator {
                    Emulator::Bigtable => "bigtable",
                    Emulator::Datastore => "datastore",
                    Emulator::Firestore => "firestore",
                    Emulator::PubSub => "pubsub",
                }.to_owned(),
                "start".to_owned()
			];
```